deploy/compose/README.md

```diff
@@ -7,8 +7,84 @@
 
 ## Setup
 
+### Setup a node for production
+
 The `docker compose` deployment is multi-faceted, allowing different combinations of tooling and extensions and different types of usage for the deployment. Please follow the guide to [set up a node](https://docs.hoprnet.org/node/node-docker-compose).
 
+### Setup a node for testing
+
+This guide will walk you through the steps required to set up a node on the Rotsee network for testing purposes.
+
+#### 1. Create an identity and make it eligible for the Rotsee network
+
+Before running a node on the Rotsee network, ensure you complete these steps: create an identity file, set up a safe, configure module instances, and link the identity to those instances.  
+
+For detailed instructions, follow [this guide](https://github.com/hoprnet/hoprnet/blob/master/hopli/README.md#create-read-identity-and-make-it-eligible-for-rotsee-network).
+
+#### 2. Download the compose folder
+
+Start by downloading the "compose" folder from the HOPR repository to your local machine: 
+
+```md
+wget https://github.com/hoprnet/hoprnet/archive/refs/heads/master.zip && unzip master.zip "hoprnet-master/deploy/compose/*" -d extracted_files && mv extracted_files/hoprnet-master/deploy/compose . && rm -rf master.zip extracted_files
```

Comment:
As is for staging, it makes more sense to put the commands of a `git clone https://github.com/hoprnet/hoprnet`, as it will be done by a developer which already has the repo